apps/wallet-mobile/src/i18n/locales/en-US.json

```diff
@@ -131,6 +131,7 @@
   "components.send.assetselectorscreen.youHave": "You have",
   "components.send.assetselectorscreen.noAssetsAddedYet": "No {fungible} added yet",
   "components.send.addressreaderqr.title": "Scan QR code address",
+  "components.send.addressreaderqr.text": "Scan recipients QR code to add a crypto address",
```

Comment:
```suggestion
  "components.send.addressreaderqr.text": "Scan recipients QR code to add a wallet address",
```

Reply:
Either way check with Product cuz the QR codes can contain the Amount for Primary. yet there is no CIP for the rest such -> memo, and secondary tokens, still it follows the URL schema so we can decode